test/e2e_node/image_volume.go

```diff
@@ -47,7 +47,7 @@ var _ = SIGDescribe("ImageVolume", feature.ImageVolume, func() {
 	const (
 		podName             = "test-pod"
 		containerName       = "test-container"
-		validImageRef       = "quay.io/crio/artifact:v1"
+		validImageRef       = "quay.io/crio/artifact:v2"
```

Comment:
Rebuilt the image using docker as regular container image. Should be now pull-able from containerd, too.

Reply:
Is it possible to reuse the existing image or add a new image to `registry.k8s.io/e2e-test-images/` instead of hardcoding the image in the test code?

- https://github.com/kubernetes/kubernetes/blob/2a32b2137236fa7a5ede1ec391f55323a3485020/test/images

Reply:
Yes, I'll change it to an already available image.